envoy/event/dispatcher.h

```diff
@@ -200,7 +200,7 @@ class Dispatcher : public DispatcherBase, public ScopeTracker {
   virtual Network::ServerConnectionPtr
   createServerConnection(Network::ConnectionSocketPtr&& socket,
                          Network::TransportSocketPtr&& transport_socket,
-                         StreamInfo::StreamInfo& stream_info) PURE;
+                         StreamInfo::StreamInfo& stream_info, Buffer::Instance* buffer) PURE;
```

Comment:
Use `Buffer::InstancePtr`, not a raw ptr.

Reply:
got it